concent_api/api-e2e-multi-threads-test.py

```diff
@@ -119,6 +128,32 @@ def send_correct_force_get_task_result(
     responses_global.append(response.__class__.__name__)
 
 
+def send_correct_force_subtask_results(
+    cluster_url: str,
+    report_computed_task: ReportComputedTask,
+    current_time: int,
+    provider_private_key: bytes,
+    requestor_private_key: bytes,
+    expected_status: Optional[int] = None,
+) -> None:
+    response = api_request(
+        cluster_url,
+        'send',
+        provider_private_key,
+        CONCENT_PUBLIC_KEY,
+        create_force_subtask_results(
+            timestamp=timestamp_to_isoformat(current_time),
+            ack_report_computed_task=create_ack_report_computed_task(
+                timestamp=timestamp_to_isoformat(current_time),
+                report_computed_task=report_computed_task,
+                requestor_private_key=requestor_private_key
+            )
+        ),
+        expected_status=expected_status,
+    )
+    responses_global.append(response.__class__.__name__)
```

Comment:
I disagree or don't understand Your intent. I moved functions using in 2 files to avoid code duplication. This is used only in this test file. Do You agree or what and why should be moved?